src/app/multicurrency-landing/multicurrency-landing.component.html

```diff
@@ -55,9 +49,9 @@ <h2>A global solution to a local problem</h2>
     <div class="b-container">
       <h2>Share this</h2>
       <div class="socials-wrapper">
-        <fa-icon [icon]="faFacebookSquare" class="social-icon"></fa-icon>
-        <fa-icon [icon]="faTwitter" class="social-icon"></fa-icon>
-        <fa-icon [icon]="faInstagram" class="social-icon"></fa-icon>
+        <a href="http://www.facebook.com/share.php?u=donate.thebiggive.org.uk/gogiveone" target="blank"><fa-icon [icon]="faFacebookSquare" class="social-icon"></fa-icon></a>
```

Comment:
@alihejazi  did you check the counterparts in `DonationCompleteComponent`? I think they are better in a few ways.

* What about protocols in the URL?
* Do we want to define share copy for platforms supporting it?
* Why are we hard-coding this when we could make it work on `thebiggive.com` (which this campaign will use!) where appropriate, as well as facilitating more realistic QA?

In this case I suggest we also check in with the team about `?utm` parameters – have we discussed with WHO how they will track traffic coming in to the page, and would it be appropriate to have a campaign parameter for these shares in line with their conventions?